src/utilities/alfalfa/AlfalfaPoint.hpp

```diff
@@ -0,0 +1,111 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
+
+#ifndef ALFALFA_POINT_HPP
+#define ALFALFA_POINT_HPP
+
+#include "AlfalfaComponent.hpp"
+#include "utilities/core/Optional.hpp"
+#include "utilities/core/Logger.hpp"
+
+#include "../UtilitiesAPI.hpp"
+
+#include <string>
+#include <json/json.h>
+
+namespace openstudio {
+namespace alfalfa {
+  namespace detail {
+    class AlfalfaPoint_Impl;
+  }
+  class UTILITIES_API AlfalfaPoint
+  {
+   public:
+    AlfalfaPoint();
```

Comment:
why is this one here? For convenience in vector or a swig thing?
or is displayName meant to be optional?